src/net/behaviour.rs

```diff
@@ -80,7 +80,12 @@ enum QueryChannel {
     GetRecord(oneshot::Sender<Result<Vec<PeerRecord>>>),
     PutRecord(oneshot::Sender<Result<()>>),
 }
-
+#[derive(Clone, Debug, PartialEq)]
+pub enum GossipEvent {
+    Subscribed(PeerId),
+    Message(PeerId, Vec<u8>),
```

Comment:
Wonder where this should be an `Arc<[u8]>`

Reply:
Good question. At least for the actyx use case we expect more data coming over the fast path. So maybe yes? But I don't have a strong opinion either way